frontend/common/src/fakeData/fakeTeams.ts

```diff
@@ -0,0 +1,68 @@
+import { faker } from "@faker-js/faker";
+// import {Team}from "../api/generated";
+
+// TO DO: Remove after #5548
```

Comment:
I believe this can be cleaned up now?

Reply:
Good catch! Removed in [adjust fake teams, remove to do](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5555/commits/9f367b98d519531b361024289ab9f29d32e600d2)